tensorflow/compiler/xla/service/gpu/ir_emission_utils.cc

```diff
@@ -319,10 +319,17 @@ llvm::Value* EmitFullWarpShuffleDown(llvm::Value* value, llvm::Value* offset,
 
   // Special case for efficiency
   if (value->getType()->isFloatTy() && bit_width == 32) {
-    return EmitCallToTargetIntrinsic(
-        TargetIntrinsicID::kShflDownF32,
-        {all_warps_mask, value, offset, builder->getInt32(kWarpSize - 1)}, {},
-        builder);
+    struct TargetFunctionCallInfo target_function_call_info(
+        TargetFunctionID::kShflDownF32);
+    // Input operands.
```

Comment:
Not sure the "Input operands.", "input types", "output type", and "Emit intrinsic or device function" comments are useful; they seem to be repeating basically what's already in the code?